doc/modules/partial_dependence.rst

```diff
@@ -158,6 +158,16 @@ PDPs. They can be plotted together with
     >>> plot_partial_dependence(clf, X, features,
     ...     kind='both')  # doctest: +SKIP
 
+If there are too many lines in an ICE plot, it can be difficult to see
+differences between individual samples and interpret the model. Centering the
+ICE at the first value on the x-axis, produces cICE plots. This puts emphasis
```

Comment:
Please add the ref right after "produces cICE plots"

Reply:
The refs are added to this doc in general, without directly citing them at any point. Should I change this style by citing in the text directly, f.i. with `..., produces cICE plots [2]_.` and adding `.. [2] A. Goldstein, A. Kapelner, ...` or how do you want me to add the ref?

Reply:
Yes that would be great.